### PR TITLE
Check users in the Following, Followers, and Subscriptions profile tabs

### LINKED ISSUE #88: Feature request: /followers /following page support
Would like the extension to work while scrolling on someone's followers/following page

### PR DESCRIPTION
**Pull request by MrAwesome**
Resolves #88

Like it says on the tin - when you're looking at a user's profile and click into Following, Followers, or Subscriptions, this will check those lists for Blue users and queue them.

(Note: this doesn't find **Subscribers** to a user, just the users they subscribe to. Obviously finding subscribers would be another nice feature, but it's outside the scope of this PR)

Also fyi, this is stacked on #119.

### CODE REVIEW

**Comment by DanielleMiu** [comment]
this all looks pretty good too, I might recommend making future PRs against the 0.3.0 llb, though. I need to get that branch updated with all the changes from 0.2.6, but any further change in js are just more things that will need to be translated to typescript in the future

**Comment by MrAwesome** [comment]
Oh! Didn't even know that existed, cool. Much rather be writing in TS
anyway. I can port both of these over to there too if you'd like.

**Comment by DanielleMiu** [comment]
> Oh! Didn't even know that existed, cool. Much rather be writing in TS anyway. I can port both of these over to there too if you'd like.

that would be awesome for this pr for sure. I'm less inclined to prevent a bugfix from going out. I still need to update the 0.3.0 pr with all the changes I did in 0.2.6, which is why it's kind of in limbo right now.

**Comment by cooljeanius** [comment]
So it appears that some merge conflicts have arisen; they'll need to be resolved...

**Comment by cooljeanius** [comment]
> > So it appears that some merge conflicts have arisen; they'll need to be resolved...
> 
> this is from before the change to typescript

Well, in any case, something that @MrAwesome will have to fix before we can merge...

**Comment by MrAwesome** [comment]
Hey y'all, been dealing with an extended family crisis so probably won't
have time or juice to deal with this for a long time - feel free to close
it up if no one else wants to take over

**Comment by rougetimelord** [comment]
I'll resolve the conflicts and bundle this with #177 tomorrow!

File: parsers/instructions.js [hunk 1]
```diff
@@ -5,6 +5,15 @@ import { BlockBlueVerified } from "../shared.js";
 // the response body. since it doesn't match one specific request, it has
 // its own file
 
+const UserTimelineInstructionPath = [
+	"data",
+	"user",
+	"result",
+	"timeline",
```
Comment (rougetimelord) [inline parsers/instructions.js:12]: @MrAwesome is the repetition here on purpose?

Comment (MrAwesome) [reply parsers/instructions.js:12]: Yes, there was a `timeline` object nested under another `timeline` - I don't remember much, but I remember that being odd. I popped into the dev console to take a quick peek just now, I think they've renamed the outer `timeline` to `timeline_v2` - but I wouldn't trust me too far on that, I'm very very out of the loop. My guess is that a lot of this code is quite stale by this point, especially given all of the... engineering changes over there :(

image

Comment (rougetimelord) [reply parsers/instructions.js:12]: Huh weird, on my end I'm still getting `{timeline: timeline: { ... } }`

**Comment by rougetimelord** [comment]
Alright transferred the functionality to #177

**Pull request closed without merging**